packages/tests/README.md

```diff
@@ -0,0 +1,234 @@
+of web test runner properties found in `wtr-config.js`.
+
+Properties specific to local and remote environments inherit and override the baseline
+configuration.
+
+Local tests use the `web-test-runner.config.js` configuration. Remote tests use
+the `web-test-runner.saucelabs.config.js` configuration.
+
+## Usage
+
+All test runs share the following env variables:
+
+```bash
+BROWSERS=<browser-type>
+CONCURRENT_BROWSERS=<number-browsers-running-at-same-time>
+CONCURRENT_FRAMES=<num-of-tests-run-in-iframes>
+```
+
+Declare the variables above to control concurrency and browser types.
+
+For example, the following env variables will run 3 tests in two browsers at the
+same time, totalling in a maximum of 6 concurrent tests.
+
+```
+BROWSERS=local
+CONCURRENT_BROWSERS=2
+CONCURRENT_FRAMES=3
+```
+
+### Run tests locally in a single browser
+
+Declare the `BROWSERS` env variable alongisde the `test` command to
+execute tests in a specific browser.
+
+The following will run tests for all packages in firefox:
+
+```bash
+BROWSERS=firefox npm run test
+```
+
+Available browser options for local tests include:
+
+- chromium
+- firefox
+- safari
+
+#### Run tests locally in multiple browsers
+
+By default, executing `npm run test` runs tests in chromium, firefox, and safari
+via Playwright.
+
+```bash
+# lit/
+npm run test
+```
+
+The command above defaults to the following env variables:
+
+```
+BROWSERS=local
+CONCURRENT_BROWSERS=3
+CONCURRENT_FRAMES=6
+```
+
+## Maintainers
+
+The following section describes testing requirements for maintainers.
+
+### Local tests
+
+Add tests for `DEV` and `PROD` modes in the new package's `package.json` file.
+
+Not all packages require `PROD` tests. If a package does not require production
+tests, use the following syntax:
+
+```JSON
+"scripts": {
+    "test": "MODE=prod npm run test:dev",
+    "test:dev": "cd ../tests && npx wtr '../<package_name>/test/**/*_test.(js|html)'",
+    "test:prod": "echo '<package_name>: prod tests are identical to dev tests'"
+}
+```
+
+If the new package requires tests in `Prod` mode, use the following syntax:
+
+```JSON
+"scripts": {
+    "test": "MODE=prod npm run test:dev",
+    "test:dev": "cd ../tests && npx wtr '../<package_name>/test/**/*_test.(js|html)'",
+    "test:prod": "MODE=prod npm run test:dev"
+}
+```
+
+### Remote tests
+
+#### Dev mode
+
+All packages should be tested remotely in `DEV` mode.
+
+Inside `web-test-runner.saucelabs.config.js` is a list called `devFiles` that provides
+web test runner test files for saucelabs.
+
+Add new package tests to this list to include them in a remote test run.
+
+```TS
+const devFiles = [
+    ...
+    '../<package_name>/test/**/*_test.(js|html)',
+];
+```
+
+#### Prod mode
+
+Not all packages run in production mode remotely.
+
+For example, packages found in [labs]('../labs/README.md') are inherently
+not meant for production and have no obligation to be tested in `PROD` mode.
```

Comment:
Note this is just de-facto; we do make production builds of the labs packages and there's nothing inherently keeping a labs package from adding a DEV_MODE debug warning or check, where we would want to test in both dev and prod -- just most haven't needed that yet. So I would err on just testing everything the same way (i.e. in both dev and prod).

I just saw this discussion: https://github.com/lit/lit/pull/2698#discussion_r844464088, and I actually get the feeling that
```
   "test:prod": "TEST_PROD_BUILD=true npm run test:dev",
```
was meant to be
```
    "test:prod": "MODE=prod npm run test:dev",
```
which is what's found in the code packages, and these packages got left behind after a variable rename or something (the `MODE` env variable is what triggers the prod/dev source remapping: https://github.com/lit/lit/blob/main/packages/tests/web-test-runner.config.js#L15-L18).

@sorvell @justinfagnani Does this match your understanding? Any reason to treat labs packages differently?

Reply:
Assuming we actually want to just run everything in dev and prod, then I think this means the `defFiles` / `prodFiles` distinction would go away.

Reply:
From blame, it looks like that's what happened: `TEST_PROD_BUILD` was renamed to `MODE` and the labs testing of prod mode got left behind: https://github.com/lit/lit/commit/c8b41b7bef2d30d3515d6942ece66fb46e967e29.

Iow I don't think it's intentional that we're not testing `labs` in prod mode.

Reply:
I see! makes sense (i was assuming a lot there)
I'll update the prod tests with `MODE` and remove any kind of "we don't run prod tests" statements

Reply:
I've added back `MODE=prod` back for labs packages, everything is running prod tests now!

Reply:
Removed all mentions of "we don't always run prod tests", it very much says "we run prod tests" now :)